lib/index.js

```diff
@@ -126,6 +155,7 @@ Ramen.prototype.identify = function(identify){
 
   // For tracking purposes
   ramenSettings._partner = 'segment.com';
+  ramenSettings.disable_location_watch = true;
```

Comment:
> In the future you might consider an anonymousId approach, where you later link identified users to an anonymousId—though that totally depends on how you want Ramen to work

Yeah we've got something in the roadmap that will allow anonymous usage, but we're waiting for some customers to clamor for it a bit more before implementing :)

> Yeah, basically just exit early when a precondition isn't met

Ok good. That's been included in the most recent commit.

> We should also update the Ramen documentation

Ok just let me know what do to @amillet89 

Thanks!